doc/lua_api.txt

```diff
@@ -5646,6 +5646,7 @@ Misc.
 * `minetest.encode_base64(string)`: returns string encoded in base64
     * Encodes a string in base64.
 * `minetest.decode_base64(string)`: returns string or nil for invalid base64
+    * Before 5.5.0 this function did not accept padding and returned nil
```

Comment:
Really? Padding was necessary to get the skinsdb updater to work, hence commit f34abaedd2.

I believe this should state 5.4.0.

Reply:
To be precise: Before 5.4 it did not accept padding and returned nil; 5.4 accepts invalid padding, which most likely results in undefined behavior.

Reply:
Right. That would fit better as a correect description.